src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -415,7 +416,7 @@ public AzureAppConfigurationOptions Map(Func<ConfigurationSetting, ValueTask<Con
 
         private static ConfigurationClientOptions GetDefaultClientOptions()
         {
-            var clientOptions = new ConfigurationClientOptions(ConfigurationClientOptions.ServiceVersion.V1_0);
+            ConfigurationClientOptions clientOptions = new ConfigurationClientOptions(ConfigurationClientOptions.ServiceVersion.V1_0);
```

Comment:
Seems unnecessary change

Reply:
I was just told that the convention was to use the type if there was an explicit type being used for the initialization. I don't mind taking it out if it's unnecessary though.

Reply:
Declare the type explicitly when you cannot tell what the type is. In this case, it's clear what the type is from the new operator.